### PR TITLE
Feat : Added watch later feature

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
User can add to watchlist playlist from explore page and video page

### CODE REVIEW

File: src/pages/SingleVideo/SingleVideo.jsx [hunk 1]
```diff
@@ -11,27 +11,51 @@ import { useUserData } from "../../context/UserDataContext";
...
+	removeWatchLaterService,
+} from "../../services/watchlist-services";
+import { actionTypes } from "../../reducers/actionTypes";
 export const SingleVideo = () => {
 	const { videoListingState } = useVideoListing();
 	const { data } = videoListingState;
 	const [opened, setOpened] = useState(false);
+	const { SET_LIKES, SET_WATCHLATER } = actionTypes;
 	const params = useParams();
 	const videoId = params.videoId;
```
Comment (msdeshmukh009) [inline src/pages/SingleVideo/SingleVideo.jsx]: `const {videoId} = useParams();`

**Review by BelideNikhil — COMMENTED** [review]
Clean code and great UI.
Do reset the like and other states on logout as it still shows liked after logging out
GTG!

**Review by msdeshmukh009 — APPROVED** [review]
This is nice and `usePlaylistUpdater` is a good abstraction. Nice work!!

**Review by BelideNikhil — APPROVED** [review]

**Review by sakshamAK — APPROVED** [review]
Great work!

**Pull request merged**